### PR TITLE
Accordion with upload button

### PR DESCRIPTION
**Pull request by akshitha-r**

### CODE REVIEW

**Review by PramodaM4444 — CHANGES_REQUESTED** [review]
Why I am seeing the changes in Example.tsx and other files in this PR where there is no changes made? It will be hard to identify the new changes. Also pls make sure the PR branch is updated when compared to the dev branch by resolving the merge conflicts.

**Comment by akshitha-r** [comment]
I have merged and now there are no conflicts with the base branch.

File: src/containers/Timesheet/Timesheet.tsx [hunk 1]
```diff
@@ -1,20 +1,68 @@
...
+                aria-controls="panel1a-content"
+                id="panel1a-header"
+                sx={{
+                    backgroundColor: (theme) =>
+                        theme.palette.mode === "light"
+                            ? theme.palette.grey[100]
+                            : theme.palette.grey[900],
+                }}
+            >
+                <Typography>ILC Details</Typography>
```
Comment (PramodaM4444) [inline src/containers/Timesheet/Timesheet.tsx]: Pls import the text from the Constants file in all the places

File: src/containers/Timesheet/Timesheet.tsx [hunk 2]
```diff
@@ -1,20 +1,68 @@
...
+            </AccordionSummary>
+            <AccordionDetails>
+                <Typography>Remarks (if any)</Typography>
+                <TextareaAutosize
+                    aria-label="minimum height"
+                    minRows={5}
+                    style={{ width: 800 }}
```
Comment (PramodaM4444) [inline src/containers/Timesheet/Timesheet.tsx]: Make use of 'rem' for CSS units

File: src/App.tsx [hunk 3]
```diff
@@ -1,25 +1,20 @@
...
+import { Login } from "./containers/Login/Login";
+import { Timesheet } from "./containers/Timesheet/Timesheet";
+
+function App() {
+    return (
+        <div>
+            <Routes>
+                <Route path="/" element={<Home />} />
+                <Route path="/login" element={<Login />} />
+                <Route path="/timesheet" element={<Timesheet />} />
```
Comment (aritraibm) [inline src/App.tsx]: I might be wrong, but are we removing existing code, could you please double-check once? thanks

File: src/containers/Timesheet/Timesheet.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import Typography from "@mui/material/Typography";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import CloudUploadIcon from "@mui/icons-material/CloudUpload";
+import { CustomButton } from "../../components/CustomButton/CustomButton";
+
+export const Timesheet: React.FC = () => (
+    <Container maxWidth="md">
+        <h4>Upload ILC</h4>
```
Comment (aritraibm) [inline src/containers/Timesheet/Timesheet.tsx]: As per our agreed coding guidelines, we are supposed to use the UI constant file for all of the static labels/ texts, pls change it, thanks

File: src/containers/Timesheet/Timesheet.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+                <Typography>ILC Details</Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+                <Typography>Remarks (if any)</Typography>
+                <TextareaAutosize
+                    aria-label="minimum height"
+                    minRows={5}
+                    style={{ width: 800 }}
+                />
+                <Box m={1} display="flex" justifyContent="flex-end">
```
Comment (aritraibm) [inline src/containers/Timesheet/Timesheet.tsx]: in-line style properties are good but sometimes it's very hard to maintain when the application is getting large. we can use the class/ styled component, thanks

File: src/containers/Timesheet/Timesheet.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+                        theme.palette.mode === "light"
+                            ? theme.palette.grey[100]
+                            : theme.palette.grey[900],
+                }}
+            >
+                <Typography>View ILC</Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+                <Typography>Approval/Rejection Remarks</Typography>
+                <TextareaAutosize
```
Comment (aritraibm) [inline src/containers/Timesheet/Timesheet.tsx]: To handle errors and other scenarios, can we create a component (like dropdown/ textbox), you can use **Textarea** which is already there. Also, from a look and feel point of view **TextareaAutosize** is not align with Material UI design (However it's a part of MUI)

**Review by aritraibm — DISMISSED** [review]

File: src/containers/Timesheet/Timesheet.tsx (hunk 5 above)
Comment (aritraibm) [reply src/containers/Timesheet/Timesheet.tsx]: I'm okay if it's a part of MUI Box API or prop

**Review by PramodaM4444 — APPROVED** [review]

**Review by ShubhamAgrawa — APPROVED** [review]

**Pull request merged**